docs/index.md

```diff
@@ -3,26 +3,30 @@ layout: page
 title: Documentation
 ---
 
-[Fluo] and [Fluo Recipes] have seperate documentation as they are different repositories with their own release cycle.
+[Apache Fluo] and [Apache Fluo Recipes] have seperate documentation as they are different repositories with their own release cycle.
 
-## Fluo
+## Apache Fluo
 
-Latest release is `{{ site.latest_fluo_release }}` made on {{ site.latest_fluo_release_date }}.
+The first release of Apache Fluo has not been made yet.
 
-* Fluo documentation - [Latest][fluo-docs-latest] \| [Archive][fluo-docs-archive]
-* Fluo API - [Latest][fluo-api-latest] \| [Archive][fluo-api-archive]
-* Fluo release summary -  [Latest][fluo-sum-latest] \| [Archive][fluo-sum-archive]
-* Fluo release notes - [Latest][fluo-notes-latest] \| [Archive][fluo-notes-archive]
+Latest non-ASF release is `{{ site.latest_fluo_release }}` made on {{ site.latest_fluo_release_date }}.
 
-## Fluo Recipes
+* Documentation - [Latest][fluo-docs-latest] \| [Archive][fluo-docs-archive]
```

Comment:
Would be nice to add a notice about non asf release to the content of the older release docs.